apps/sms/js/thread_ui.js

```diff
@@ -2531,10 +2531,10 @@ var ThreadUI = {
       // eg. type "a", then delete it.
       // Always remove the the existing results.
       this.toggleRecipientSuggestions();
-      return;
+      return Promise.resolve(null);
```

Comment:
> ; do you think we should just not call searchContact in case there is no filterValue, and remove the suggestion panel earlier in the code?

Yep, something like this sounds reasonable to me